apache/superset
# chore: Eslint custom plugin to warn about hex and literal colors

### SUMMARY
Warns about the usage of rgba, hex and literal colors over the preferred theme variables.

### TESTING INSTRUCTIONS
1. Launch `npm run _lint` in superset-frontend
2. Check all the warnings

### ADDITIONAL INFORMATION
